### DISCUSSION TITLE
Avoid all questions from glab when running it in a script

### DISCUSSION THREAD

**Discussion by frodr1k** [profclems/glab#805] [Q&A] [answered]
We are trying to incoperate "glab" into a powershell-script (on Windows). Most of it we've managed to figure out but there are some caveats left until we feel it's finnished.

We want to predefine wirth command line options or avoid questions that stops the script execution. Anyone have an idea how avoid these questions? 

I've tried to figure out a way by reading the manual but.... :-)

**Comment by profclems** [comment]
You can disable the prompt with the `NO_PROMPT` or `PROMP_DISABLED` env variable or directly in the config
```
export NO_PROMPT=1
```
or disable it in the config
```
glab config set -g no_prompt true
```

**Reply by frodr1k** [reply]
Thanks!  I tried it though but no succes

D:\Script\MMMMMM\Glab\testing>..\glab repo create ZZZ/ddd -d "Autogenerated by Script"
✓ Created repository ZZZ / ddd on GitLab: https://git.XXX.yy/ZZZ/ddd
? Create a local project directory for ZZZ / ddd? (Y/n)

**Reply by profclems** [reply]
Oops, I just realised that the repo create command does not check if prompt is disabled. I will pull up a fix for that. 
Aside that, most of the commands should respect the `NO_PROMPT` or `PROMPT_DISABLED` env variables.

**Reply by frodr1k** [reply]
Ah! that explains it. We'll sit tight for the fix then ;)

**Reply by profclems** [reply]
This PR #811 should solve this. I may create a new release today with that

**Comment by frodr1k** [comment, marked as answer]
Thanks for the fix! Seems to work now! You saved our day! ... So ... where do I send the beer?

**Reply by profclems** [reply]
Glad to be of help!
Our collective is always open: https://opencollective.com/glab
